src/engraving/rw/inoutdata.h

```diff
@@ -48,6 +48,7 @@ struct ReadInOutData {
 
     // out
     SettingsCompat settingsCompat;
+    std::optional<double> spatium = std::nullopt;
```

Comment:
this is `in` params
I think better make some other name, maybe `overridedSpatium`

Reply:
Done